sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/QueryPlan.scala

```diff
@@ -189,9 +189,16 @@ abstract class QueryPlan[PlanType <: QueryPlan[PlanType]] extends TreeNode[PlanT
     val codegenIdStr =
       getTagValue(QueryPlan.CODEGEN_ID_TAG).map(id => s"[codegen id : $id]").getOrElse("")
     val operatorId = getTagValue(QueryPlan.OP_ID_TAG).map(id => s"$id").getOrElse("unknown")
-    s"""
+    val argumentString = argString(SQLConf.get.maxToStringFields)
+
+    val result = s"""
        |($operatorId) $nodeName $codegenIdStr
      """.stripMargin
+    if (argumentString != null && !argumentString.isEmpty) {
+      s"""${result} |Arguments: $argumentString\n""".stripMargin
```

Comment:
This is too hard to read. How about
```
val result = s"($operatorId) $nodeName $codegenIdStr\n"
if (argumentString.nonEmpty) {
  result + s"Arguments: $argumentString\n"
} else {
  result
}
```

Reply:
I see, thanks!

Reply:
Oh, I just remember why I tried so complicated writing here. 
`result + s"Arguments: $argumentString\n"`  will lead to unexpected padding before Arguments. 
Thus to avoid the padding and not using improper multiline string maybe we can use 
`s"${result} |Arguments: $argumentString\n".stripMargin` ?

Any other suggestions? Thanks @cloud-fan

Reply:
how about `result.trim + s"Arguments: $argumentString\n"`?

Reply:
It seems trim will break the first line padding of result.

`result.trim + s"\nArguments: $argumentString\n"`
Shows
```
     (9) BroadcastExchange 
Input [2]: [key#x, val#x]
Arguments: HashedRelationBroadcastMode(List(cast(input[0, int, true] as bigint))), [id=#x]
```

Reply:
how about
```
val baseStr = s"($operatorId) $nodeName $codegenIdStr"
if (argumentString.nonEmpty) {
  s"""
    |$baseStr
    |Arguments: $argumentString
  """. stripMargin
} else {
  s"""
  |$baseStr
  """. stripMargin
}
```

Reply:
Yea, it works with better code style. Thanks a lot!